docs/essentials/introduction.md

```diff
@@ -11,6 +11,26 @@ A major strength of Storybook are [addons](/addons/) that extend Storybook’s U
 - [Backgrounds](./backgrounds.md)
 - [Toolbars & globals](./toolbars-and-globals.md)
 
+<details>
+<summary>Upgraders may need to install and configure essential addons manually.</summary>
+
+Install the addon-essentials package from `npm`.
+
+```shell
+npm install --save-dev @storybook/addon-essentials
+```
+
+Then configure Storybook to use it by modifying the `main.js` configuration file.
+
+```js
+// .storybook/main.js
+
+module.exports = {
+  addons: ['@storybook/addon-essentials'],
+};
+```
+</details>
+
```

Comment:
@awayken sorry for the delay in checking this. Going about it we can probably make this into it's own section of the documentation. Instead of a details item, that is going to be collapsed and place it after "Disabling addons".

We could probably go with:

```md
### Upgrading from previous versions

If you're upgrading from a previous Storybook version, you will need to add the `@storybook/addon-essentials` package manually.

In your terminal run the following command:

The install command here.


<div class="aside"
If you're using <a href="https://yarnpkg.com/">yarn</a>, you'll need to adjust the command accordingly.
</div>

Update your Storybook configuration (in `.storybook/main.js`) to include the essentials addon.

Insert config snippet here
```

Let me know what you think

Reply:
That sounds good to me. Makes the content more discoverable by not being collapsed and also keeps it from disrupting the flow at the top, especially since it's likely that most users will not need this information.

I've updated the PR with these changes. Let me know what you think.